tests/system/model_monitoring/assets/application.py

```diff
@@ -114,3 +114,14 @@ def do_tracking(
 
 class NoCheckDemoMonitoringApp(DemoMonitoringApp, check_num_events=False):
     pass
+
+
+class ErrApp(ModelMonitoringApplicationBaseV2):
+    NAME = "err-app"
+
+    def do_tracking(
+        self,
+        monitoring_context: mm_context.MonitoringApplicationContext,
+    ) -> list[mm_results.ModelMonitoringApplicationResult]:
+        monitoring_context.logger.info("Running demo app")
```

Comment:
Okay to skip, but I think it's better to differentiate from the other demo app log:
```suggestion
        monitoring_context.logger.info("Running error app")
```